src/main/java/Factory/DaoFactories/ItemDaoFactory.java

```diff
@@ -0,0 +1,16 @@
+package Factory.DaoFactories;
+
+import dao.Implementations.ItemDaoHibernateImpl;
+import dao.Interfaces.ItemDao;
+
+public class ItemDaoFactory {
```

Comment:
in this class we can create singleton 
and then we can remove creation of singleton from class getItemDaoHibernateImpl

do the same in other factories